app/controllers/idv/link_sent_poll_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Idv
-  class CaptureDocStatusController < ApplicationController
+  class LinkSentPollController < ApplicationController
```

Comment:
Note from the filename at the top that this (and the test) are also renamed. GitHub doesn't make this super obvious.